core/ir/riscv64/codec.py

```diff
@@ -872,6 +872,9 @@ def __fixup_uncompressed_inst(self, inst: Instruction):
             elif rs3 == 0x3: # SC.W/D instructions
                 dbg(f'fixup: {inst.name} {[f.name for f in inst.flds]}')
                 inst.flds[2] = Field.V_S_RS1_DISP
+                # Swap the rd and mem operand positions so that mem becomes the
+                # first operand to be consistent with AArch64.
+                inst.flds[2], inst.flds[3] = inst.flds[3], inst.flds[2]
```

Comment:
Are there other load/store instructions that need this?

Should this be done in the table/database source, instead of here?  Or that order is the bit position order?

Reply:
> Are there other load/store instructions that need this?

This is the only instruction that has both a register destination and a memory destination. Well, it's also true for "read-modify-write" atomic instructions like `amoswap`, but their memory destination is implicit, so it's not the same case.

> Should this be done in the table/database source, instead of here? Or that order is the bit position order?

Yes, the operand order in the txt files is the bit position order, but it's not used by our codegen, so I think both ways are acceptable, I'll keep it this way.